graphql/tests/task/taskLogs/results.json

```diff
@@ -8,11 +8,11 @@
             "taskLogs": {
               "systemLogs": [
                 {
-                  "type": "S",
+                  "type": "",
                   "severity": "I",
                   "message": "    [javac]     protected void finalize() {",
                   "timestamp": "2019-12-07T13:31:19.393-05:00",
-                  "version": 1
+                  "version": 0
```

Comment:
What is the motivation behind these changes?

Reply:
That was the DB version of the logs, which we no longer support.